src/bridge/ui_commands.rs

```diff
@@ -102,6 +107,11 @@ impl SerialCommand {
                 .await
                 .expect("Mouse Drag Failed");
             }
+            SerialCommand::Paste { data, crlf, phase } => {
+                nvim.paste(data.as_ref(), crlf, phase)
+                    .await
+                    .unwrap_or(false);
```

Comment:
Skip error when paste in `modifiable=off buffer` (read-only files, startup screens and etc). We can check `modifiable` var directly and don't call paste, but it's shorter to just ignore error.